modules/configuring-the-fine-tuning-job.adoc

```diff
@@ -0,0 +1,312 @@
+:_module-type: PROCEDURE
+
+[id="configuring-the-fine-tuning-job_{context}"]
+= Configuring the fine-tuning job
+
+[role='_abstract']
+Before you can use a training job to fine-tune a model, you must configure the training job.
+You must set the training parameters, define the training function, and configure the Kubeflow Training Operator (KFTO) SDK.
+
+
+.Prerequisites
+
+* You can access an OpenShift cluster that has sufficient worker nodes with supported accelerators to run your training or tuning job.
+
+ifndef::upstream[]
+* You can access a workbench that is suitable for distributed training, as described in link:{rhoaidocshome}{default-format-url}/working_with_distributed_workloads/preparing-the-distributed-training-environment_distributed-workloads#creating-a-workbench-for-distributed-training_distributed-workloads[Creating a workbench for distributed training].
+endif::[]
+ifdef::upstream[]
+* You can access a workbench that is suitable for distributed training, as described in link:{odhdocshome}/working-with-distributed-workloads/#creating-a-workbench-for-distributed-training_distributed-workloads[Creating a workbench for distributed training].
+endif::[]
+
+
+* You have Admin access for the data science project.
+** If you created the project, you automatically have Admin access. 
+** If you did not create the project, your cluster administrator must give you Admin access.
+
+
+.Procedure
+. Open the workbench, as follows:
+.. Log in to the {productname-long} web console.
+.. Click *Data science projects* and click your project.
+.. Click the *Workbenches* tab. 
+If your workbench is not already running, start the workbench.
+.. Click the *Open* link to open the IDE in a new window. 
+
+. Click *File -> New -> Notebook*.
+
+. Install any additional packages that are needed to run the training or tuning job.
```

Comment:
The finetuning code below should be meant as an example only. Customer can have completely different workbench content.

I think we should mention somewhere that this is just an example usage (unless I overlooked it somewhere).

Reply:
For the example below customer needs to create a PersistentVolumeClaim with name `shared` and ReadWriteMany access mode (RWX).